sf-fx-runtime-java-logger/src/main/java/com/salesforce/functions/jvm/runtime/logger/DefaultLoggingFormatter.java

```diff
@@ -23,17 +25,32 @@ public DefaultLoggingFormatter(Clock clock) {
   public String format(String loggerName, Level level, String message) {
     String invocationId = MDC.get("function-invocation-id");
     String localDateTimeString = LocalDateTime.now(clock).format(DATE_TIME_FORMATTER);
+    String zoneId = clock.getZone().toString();
+    Map<String, String> stringFields =
+        new HashMap<String, String>() {
+          {
+            put(zoneId, localDateTimeString);
```

Comment:
I think this is a misunderstanding. I think we should just use ISO 8601 with a TZ identifier ([ISO_OFFSET_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_OFFSET_DATE_TIME)) for the datetime string and use a static key for it in the log message. 

If we use a dynamic key, grepping logs would become very hard since the key might differ between machines. Also, even when in the same timezone (as in offset to UTC), a different zone id might be returned. It could be a geographical id like `Europe/Paris` or `Europe/Berlin` (both referring to CET/CEST depending on DST).